frontend/src/router.js

```diff
@@ -36,10 +37,14 @@ const router = (
               <Route exact path="/enrollment" component={Enroll} />
               <Route exact path="/meet-the-team" component={Team} />
               <Route exact path="/ambassadors" component={Ambassadors} />
-              <Route exact path="/events" component={Events} />
+              {/* <Route exact path="/events" component={Events} /> */}
               <Route exact path="/admin" component={Admin} />
-              <Route exact path="/login" component={Login} />
-              <Route exact path="/resources" component={Resources} />
+              {/* <Route exact path="/login" component={Login} /> */}
```

Comment:
This should not be commented out.

Reply:
I commented out because it didn't have files or i couldnt find where it  was coming from.